packages/api/src/routers/blob/getCount.ts

```diff
@@ -19,52 +19,74 @@ export async function countBlobs(
   prisma: BlobscanPrismaClient,
```

Comment:
Since this function is now more complex and we use the precalculated values to improve the performance... can you add a comment to this function explaining that? 

Also adding some test cases for this function would be helpful to check that the returned values didn't change.

Reply:
Done